Assets/Scripts/Layers/LayerTypes/RootLayer.cs

```diff
@@ -1,29 +1,28 @@
 using System;
-using System.Collections;
 using System.Collections.Generic;
 using System.Linq;
-using Netherlands3D.Twin.UI.LayerInspector;
+using Netherlands3D.Twin.Projects;
 using Newtonsoft.Json;
 using UnityEngine;
 
 namespace Netherlands3D.Twin.Layers
 {
     [Serializable]
-    public class RootLayer : LayerNL3DBase //todo: make an extension of base and make functions protected
+    public class RootLayer : LayerNL3DBase
     {
         [JsonIgnore] public List<LayerNL3DBase> SelectedLayers { get; private set; } = new();
 
         public RootLayer(string name) : base(name)
         {
         }
 
-        public void AddLayerToSelection(LayerNL3DBase layer) //todo: make protected once this is an extension of base
+        public void AddLayerToSelection(LayerNL3DBase layer)
         {
             if (!SelectedLayers.Contains(layer))
                 SelectedLayers.Add(layer);
         }
 
-        public void RemoveLayerFromSelection(LayerNL3DBase layer) //todo: make protected once this is an extension of base
+        public void RemoveLayerFromSelection(LayerNL3DBase layer)
         {
             if (SelectedLayers.Contains(layer))
```

Comment:
Pro-tip: you do not need to use Contains before doing a remove; remove returns a bool whether it worked and you can call it directly and perform the rest of your clean up if Remove returned true

Reply:
Thanks! I didn't know that, I removed it